opal/runtime/opal_progress.h

```diff
@@ -1,34 +1,26 @@
-/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */ /*
-                                                                * Copyright (c) 2004-2005 The
-                                                                * Trustees of Indiana University and
-                                                                * Indiana University Research and
-                                                                * Technology Corporation.  All
-                                                                * rights reserved. Copyright (c)
-                                                                * 2004-2006 The University of
-                                                                * Tennessee and The University of
-                                                                * Tennessee Research Foundation. All
-                                                                * rights reserved. Copyright (c)
-                                                                * 2004-2005 High Performance
-                                                                * Computing Center Stuttgart,
-                                                                *                         University
-                                                                * of Stuttgart.  All rights
-                                                                * reserved. Copyright (c) 2004-2005
-                                                                * The Regents of the University of
-                                                                * California. All rights reserved.
-                                                                * Copyright (c) 2006-2014 Los Alamos
-                                                                * National Security, LLC.  All
-                                                                * rights reserved. Copyright (c)
-                                                                * 2018      Triad National Security,
-                                                                * LLC. All rights reserved.
-                                                                *
-                                                                * Copyright (c) 2020      Intel,
-                                                                * Inc.  All rights reserved.
-                                                                * $COPYRIGHT$
-                                                                *
-                                                                * Additional copyrights may follow
-                                                                *
-                                                                * $HEADER$
-                                                                */
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- *//*
```

Comment:
Maybe put the second `/*` on a new line to prevent scrambling in the future.

Reply:
Good catch - thanks. Updated.